app.js

```diff
@@ -0,0 +1,187 @@
+const playerController = (() => {
+  class Player {
+    constructor(name, symbol) {
+      this.name = name;
+      this.moves = [];
+      this.symbol = symbol;
+      this.winner = false;
+      this.active = false;
+    }
+  }
+
+  return {
+    addPlayer(name, symbol) {
+      return new Player(name, symbol);
+    },
+    addMove(move) {
+      this.moves.push(move);
+    },
+
+    addWinener() {
+      this.winner = true;
+    },
+
+    activePlayer(active) {
+      this.active = active;
+    },
+  };
+})();
+
+const gameBoardController = ((playerController) => {
+  class GameBoard {
+    constructor(xPlayer, oPlayer) {
+      this.xPlayer = xPlayer;
+      this.oPlayer = oPlayer;
+      this.status = "Running";
+      this.currentPlay = xPlayer;
+      this.winner = null;
+      this.playCount = 0;
+    }
+  }
+
+  const showWinner = (game) => {
+    document.getElementById("winner").innerHTML = game.currentPlay.name === game.oPlayer.name
+      ? game.xPlayer.name
+      : game.oPlayer.name;
+
+    document.getElementById("message").style.display = "block";
+    document.getElementById("game-status").innerHTML = "status = game over";
+    game.status = "Game Over";
+    document.getElementById("reset").classList = "";
+  };
```

Comment:
[OPTIONAL]
As mentioned by @bolah2009, you could remove the "Now you turn !" text when the game ends, that is when a player wins or when there are no more possible moves to make and there's no winner (a tie)

Reply:
Its fixed now